### PR TITLE
Add option --webdewey and guess output format from filename

### PR DESCRIPTION
**Pull request by nichtich**

### CODE REVIEW

**Comment by nichtich** [comment]
I also extended the README in branch https://github.com/gbv/mc2skos/tree/readme but not mentioning option `--webdewey` yet.

File: mc2skos/mc2skos.py [hunk 1]
```diff
@@ -239,7 +241,13 @@ def main():
         return
 
     supported_formats = ['turtle', 'jskos', 'ndjson']
-    if args.outformat not in supported_formats:
+    if not args.outformat and args.outfile:
+        ext = args.outfile.rpartition('.')[-1]
```
Comment (danmichaelo) [inline mc2skos/mc2skos.py]: Interesting, I didn't know about the `rpartition` method, but it's quite useful here as it returns a 3-tupple even if there's no separator in the string :) There's also `os.path.splitext`

**Comment by danmichaelo** [comment]
Thanks! I can also document the `--webdewey` option.

**Comment by danmichaelo** [comment]
Btw. the `--notes` flags is mostly included because I'm not allowed to publish that for WebDewey. But for a general tool it probably makes more sense to include notes by default and have a `--exclude-notes` flag for the WebDewey case.

**Comment by nichtich** [comment]
Thanks! I'd prefer `--no-notes` instead of `--exclude_notes` but nevermind.

**Pull request merged**